src/extension/background.js

```diff
@@ -337,6 +337,15 @@ async function updateHelpContent() {
       log.info(`sidekick now ${display ? 'shown' : 'hidden'}`);
     }
   });
+
+  // listen for add/remove project calls from the install helper
+  chrome.runtime.onMessage.addListener(({ action: actionFromTab }, { tab }) => {
+    // check if message contains action and is sent from right tab
+    if (tab && tab.url && new URL(tab.url).pathname.startsWith(SHARE_PREFIX)
+      && actionFromTab && typeof actions[actionFromTab] === 'function') {
+      actions[actionFromTab](tab);
+    }
+  });
```

Comment:
maybe wait and send response?

```suggestion
  // listen for add/remove project calls from the install helper
  chrome.runtime.onMessage.addListener(async ({ action: actionFromTab }, { tab }, sendResponse) => {
    // check if message contains action and is sent from right tab
    if (tab && tab.url && new URL(tab.url).pathname.startsWith(SHARE_PREFIX)
      && actionFromTab && typeof actions[actionFromTab] === 'function') {
      await actions[actionFromTab](tab);
      await sendResponse('ok');
    }
  });
```

Reply:
To do what?

Reply:
`window.location.reload()`, but if it's not needed anymore, that's ok.